server/routes/api.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const axios = require('axios')
+const City = require("../models/city")
+
+const router = express.Router()
+
+router.get('/city/:city', async function (req, res) {
+	let city = req.params.city
+
+	let apiRequest = await axios.get(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=e2522630235d331673040866c32bdf37&units=metric`)
+	let data = apiRequest.data
+
+	res.send(data)
+})
+
+router.get('/cities', async function (req, res) {
+
+	let findCities = await City.find({})
+	res.send(findCities)
+})
+
+router.post('/city', function (req, res) {
+	let cityNameObj = req.body
+	let cityInstance = new City(cityNameObj)
+
+	cityInstance.save()
```

Comment:
Make sure you wait for until the `save` operation finishes before you respond with a success. `save` is asynchronous and this is important because if there is an error in the DB you don't want to mislead the client. You can use `async/await`.